File: sem2_c_cpp_java/Assignment6_14_02_23/SampleCodes_Inheritance_Costructor_CPP_14_2_23/CPP/MLI1.cpp
#include<iostream>
using namespace std;

class A
{
	public:
		void f1()
		{
			cout << "HEllo" << endl;
		}
};

class B : public A
{
	public:
		void f2()
		{
			cout << "Hi" << endl;

		}
};

class C : public B
{
	public:
		void f3()
		{
			cout << "Hi from C" << endl;

		}
};
int main()
{
	class C c1;
	c1.f1();
	c1.f2();
	c1.f3();
}

File: sem2_c_cpp_java/Assignment6_14_02_23/SampleCodes_Inheritance_Java_14_2_23/Java/MLI1.java
import java.util.*;

class A1
{
	public void f1()
	{
		System.out.println("Hello");
	}

};

class B1 extends A1
	{
		public void f2()
		{
			System.out.println("Hello from B1");
		}
		public static void main(String args[])
		{
			C1 c1 = new C1();
			c1.f1();
			c1.f2();
			c1.f3();
		}
	};

class C1 extends B1
{
	public void f3()
	{
		System.out.println("Hello from C1");
	}
};


